content/moderation/moderation-overview.md

```diff
@@ -0,0 +1,324 @@
+the [mod log](#mod-channels) and is also, by default, DM'd to the offending user when using [certain
+commands](#moderation-dms). This reason may be retroactively edited using the [Reason Command](/commands#reason).
+
+Each of the main moderation tools have individual configuration pages, accessible through the tabs (**13**).
+
+### Common Settings
+
+![Common Moderation Configuration](common_moderation_settings.png)
+
+<center>
+
+**1** Permitted Roles **2** Reason Optional Toggle **3** Moderation DM
+
+</center>
+
+There are common configuration options worth mentioning.
+
+#### Permitted Roles
+
+By default, any user with the relevant Discord permission in your server (timeout members for the timeout command, kick
+members for the kick command, etc.) are able to run the corresponding moderation command. Selecting permitted roles
+(**1**) will allow said roles to run the moderation commands **even if** they do not have required Discord permissions.
+
+Give/Takerole Permitted Roles (**8**) operate in the same way.
+
+#### Reason Optional
+
+By default, Timeout, (Un)Mute, Kick, and Ban tools require the moderator to specify a reason. This forces your staff
+(and yourself, no exceptions!) to specify a reason for taking mod action for the mod logs, and also by default to be
+DM'd to the offending user.
+
+To remove the requirement and make the reason optional, you may toggle this on (**2**).
+
+#### Moderation DMs
+
+Moderation DMs are template scripts (**3**) which are executed within the offending user's DM. Text output to the response will
+be directly sent to the offending user.
+
+{{% button href="/reference/templates" style="transparent" %}}Templates Reference{{% /button %}}
+
+Additional template dot context data is available for Moderation Actions:
+
+- `{{.Reason}}`: The reason specified in the timeout.
+- `{{.Author}}`: The author of the punishment, is a [user object](/reference/templates#user).
+- `{{.Duration}}`: The duration of the mod action in [time.Duration format](/reference/templates#time).
+- `{{.HumanDuration}}`: The duration in a human friendly format (`1 hour and 3 minutes` for example).
+- `{{.WarningID}}`: The ID of the warning (when using the warn command).
+
+Any errors which occur in the template execution will not be logged by default. The offending user will only receive a
+DM from your server saying **Failed executing template.** Setting a [Moderation DM Error channel](#mod-channels) allows
+for the more detailed errors to be captured.
+
+Moderation actions which send DMs are:
+
+- Timeout
+- Mute
+- Kick
+- Ban
+- Warn
+
+### Timeout
+
+Timing out a member employs Discord's timeout feature to prevent users from interacting with your server. Discord's
+timeout temporarily removes the member's permissions to send messages, add reactions, and join voice channels,
+indiscriminantly and regardless of roles or overrides. Using a bot to timeout a member allows you to assign a specific
```

Comment:
```suggestion
Timing out a member employs Discord's timeout feature to prevent users from interacting with your server. Discord's
timeout temporarily removes the member's permissions to send messages, add reactions, and join voice channels,
regardless of roles or overrides. Using a bot to timeout a member allows you to assign a specific
```

Reply:
this part needs to be changed anyways, as i realized ~~when lotnik timed me out~~ that you lose more than just these three permissions. I'm struggling to find Discord documentation about timeout but my working hypothesis is all permissions except read and message history are removed.

Reply:
> Timed out members will temporarily lose all permissions except `VIEW_CHANNEL` and `READ_MESSAGE_HISTORY`. Owners and admin users with `ADMINISTRATOR` permissions are exempt.
https://discord.com/developers/docs/topics/permissions#permissions-for-timed-out-members

Reply:
I think condensing this to "removes all permissions except to read
messages" suffices — `VIEW_CHANNEL` and `READ_MESSAGE_HISTORY` are,
after all, the required combination to read messages.

We could potentially also get away with removing the note about
administrators and owners being exempt from this, as it's clearly stated
that `ADMINISTRATOR` (the permission) overrides anything else.